docker-compose.yml

```diff
@@ -23,9 +23,13 @@ x-env-mapping: &env
 services:
   worker: &worker
     <<: *env
-    image: addons/addons-server:latest
+    build: .
```

Comment:
As mentioned on IRC, we don't want to keep that, because this forces a local build instead of a pull. If someone wants to locally build an image, they should either use a custom file if they want to use `docker-compose`, or just use regular `docker`.